kustomize/base/shard/kustomization.yaml

```diff
@@ -6,4 +6,5 @@ resources:
   - resources/bridgeingresses.com.redhat.service.bridge-v1.yml
   - resources/bridgeexecutors.com.redhat.service.bridge-v1.yml
   - resources/managedbridges.com.redhat.service.smartevents-v1.yml
+  - resources/managedprocessors.com.redhat.service.smartevents-v1.yml
```

Comment:
It seems also related to v1 API.

Reply:
Likewise.. the `managedXXX` tells us it is for RHOSE `v2`.

Reply:
Same as above